### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the combo boxes in "Run on web App for Containers" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1465]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on  web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.->create new radio button
6. Tab to Container Registry combo box
    Subscription combo box 
   "use Existing" combo box under resource group expand button.
   "Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
   "Java Version and Artifact" combo boxes under "Java" expand button.
7. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA is not announcing the name of the combo boxes

**Jaws behaviour:**
combo box closed
blank
1 of 1
To change the selection use arrow keys

**NVDA Behaviour:**
combo box  collapsed
combo box  collapsed
combo box  collapsed

**Expected Result:**
Jaws and NVDA should announce the name of the combo boxes
Name should be 
Container Registry
Subscription
"use Existing" combo box under resource group expand button.
 "Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
 "Java Version and Artifact" combo boxes under "Java" expand button.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for container-com bo box-name -empty](https://user-images.githubusercontent.com/22956130/36102175-8b7a441c-1031-11e8-9367-9944ccbeb204.jpg)
![cdb1bf32](https://user-images.githubusercontent.com/22956130/36102189-9376dda6-1031-11e8-9638-8337e891eb7e.jpg)
![run on web app for containers-combo box-explorer](https://user-images.githubusercontent.com/22956130/36102192-96a5a8f4-1031-11e8-9cea-70b4f498ff1c.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749927
